src/components/view/view.api.json

```diff
@@ -24,5 +24,6 @@
     {"name": "backgroundColor", "type": "string", "description": "Set background color"},
     {"name": "recorderTag", "type": "'mask' | 'unmask'", "description": "Recorder Tag"},
     {"name": "style", "type": "ViewStyle", "description": "Custom style"}
-  ]
+  ],
+  "snippet": ["<View style={{width: 50, height: 50}} bg-orange50/>"]
```

Comment:
I played with and it's really fun, I make a more interesting snippet, let's use it (: 
```
<View row gap-s5 centerV>
  <View style={{width: 200, height: 200}} bg-purple40 centerH>
   <View style={{width: 60, height: 60}} bg-green20/>
  </View>
  <View style={{width: 150, height: 150}} bg-orange30 bottom right>
   <View style={{width: 50, height: 50}} bg-yellow40 br100 margin-s2/>
  </View>
</View>
```